velox/dwio/common/ScanSpec.h

```diff
@@ -358,7 +358,10 @@ class ScanSpec {
   /// Apply filter to the input `vector' and set the passed bits in `result'.
   /// This method is used by non-selective reader and delta update, so it
   /// ignores the filterDisabled_ state.
-  void applyFilter(const BaseVector& vector, uint64_t* result) const;
+  void applyFilter(
+      const BaseVector& vector,
+      vector_size_t size,
```

Comment:
nit: can you update the description to add context about size and why it is necessary